src/deploy/campaign/environments.ts

```diff
@@ -113,8 +127,9 @@ export const getConfig = async ({
       defaultRoyaltyFraction: royaltyFraction,
     },
     rootPriceConfig: priceConfig,
+    zTokenConfig: zConfig,
     zeroVaultAddress: zeroVaultAddressConf as string,
-    mockMeowToken: process.env.MOCK_MEOW_TOKEN === "true",
+    mockZToken: process.env.MOCK_Z_TOKEN === "true",
```

Comment:
It would make more sense to set this value in the above on L96 that you added where you already check `process.env.MOCK_Z_TOKEN`